lib/bridge/strain.rb

```diff
@@ -36,6 +36,21 @@ def self.minors
       [Club, Diamond]
     end
 
+    @mapping = {}
+    %w(Clubs Diamonds Hearts Spades NoTrumps
```

Comment:
@danielevans Clean Code.
I've just started The Pragmatic Programmer as well.
Did a lot of studying of Ruby today as well, trying to comprehend method lookup and the inheritance chain, and a few talks from Jim Weirich.